engines/ags/ags.cpp

```diff
@@ -386,7 +386,11 @@ SaveStateList AGSEngine::listSaves() const {
 }
 
 void AGSEngine::setGraphicsMode(size_t w, size_t h) {
+#ifdef USE_OPENGL
+	Graphics::PixelFormat FORMAT(4, 8, 8, 8, 8, 0, 8, 16, 24);
```

Comment:
Maybe add a comment here that the format isn't correct and colors don't match? At least it's what I understood from your previous PR.

Reply:
I am not sure about including the pixel format change here since this is not a proper fix but rather a quick hack to allow the games to start (with wrong colours) if I understand properly? Ideally we would use `System::getSupportedFormats()` and do format conversions where needed. Maybe include the change as a separate commit if you want to include it since this is unrelated to the `char` -> `int8` change?

Reply:
Also this change uses a compile time define, which means it also changes the format when using the surface sdl graphics manager on SDL ports (and I guess break the coolers there). Ideally if you want to include such a change it should only be done when the OpenGL graphics manager is used (so as a runtime check rather than `#define`).

Reply:
Will open a new PR for fixing the Color Formats.